### ISSUE TITLE
"Password should be longer" should be displayed to the user while trying to register a new membership

### ISSUE THREAD

**Issue by gomezvillegasdaniel** [freeCodeCamp/pantry-for-good#364]
https://pantry-for-good.herokuapp.com/users/signup
![image](https://user-images.githubusercontent.com/6345953/37245592-1300e9da-2460-11e8-9e82-eab203662f93.png)

**Comment by gomezvillegasdaniel** [comment]
working on this @vanessasena, is it ok?

**Comment by CraigBanach** [comment]
In the message, you should put the exact requirements for the user so they don't have to guess how long it needs to be.

e.g. "Your password needs to be between 8 and 36 characters"

**Comment by gomezvillegasdaniel** [comment]
@CraigBanach Yeah, I do have that in mind, but indeed, it accepts at least 7 chararacters
